apis/logging/v1/clusterlogging_types.go

```diff
@@ -184,6 +188,24 @@ type ElasticsearchSpec struct {
 	ProxySpec `json:"proxy,omitempty"`
 }
 
+type LokiStackSpec struct {
```

Comment:
The `LokiStackSpec` should require an explicit user choice of the `StorageClass` used for the PVC:
```go
	// Storage class name defines the storage class for ingester/querier PVCs.
	//
	// +required
	// +kubebuilder:validation:Required
	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:io.kubernetes:StorageClass",displayName="Storage Class Name"
	StorageClassName string `json:"storageClassName"`
````